Library/Homebrew/sorbet/files.yaml

```diff
@@ -880,6 +880,30 @@ true:
   - ./locale.rb
   - ./metafiles.rb
   - ./official_taps.rb
+  - ./os/mac/pkgconfig/10.16/expat.pc
+  - ./os/mac/pkgconfig/10.16/libcurl.pc
+  - ./os/mac/pkgconfig/10.16/libedit.pc
+  - ./os/mac/pkgconfig/10.16/libexslt.pc
+  - ./os/mac/pkgconfig/10.16/libffi.pc
+  - ./os/mac/pkgconfig/10.16/libxml-2.0.pc
+  - ./os/mac/pkgconfig/10.16/libxslt.pc
+  - ./os/mac/pkgconfig/10.16/ncurses.pc
+  - ./os/mac/pkgconfig/10.16/ncursesw.pc
+  - ./os/mac/pkgconfig/10.16/sqlite3.pc
+  - ./os/mac/pkgconfig/10.16/uuid.pc
+  - ./os/mac/pkgconfig/10.16/zlib.pc
+  - ./os/mac/pkgconfig/11.0/expat.pc
+  - ./os/mac/pkgconfig/11.0/libcurl.pc
+  - ./os/mac/pkgconfig/11.0/libedit.pc
+  - ./os/mac/pkgconfig/11.0/libexslt.pc
+  - ./os/mac/pkgconfig/11.0/libffi.pc
+  - ./os/mac/pkgconfig/11.0/libxml-2.0.pc
+  - ./os/mac/pkgconfig/11.0/libxslt.pc
+  - ./os/mac/pkgconfig/11.0/ncurses.pc
+  - ./os/mac/pkgconfig/11.0/ncursesw.pc
+  - ./os/mac/pkgconfig/11.0/sqlite3.pc
+  - ./os/mac/pkgconfig/11.0/uuid.pc
+  - ./os/mac/pkgconfig/11.0/zlib.pc
```

Comment:
I've not been keeping up so maybe I've misunderstood what this does: I was under the impression that `files.yaml` was for Ruby files. None of the other `.pc` files are listed here.

Reply:
You're correct! Sorry that I missed this. Updating right now.